docs/source/index.rst

```diff
@@ -36,12 +41,12 @@ These documentation topics build on the previous set to include help with
 designing new experiments for others to use.
 
 .. toctree::
+    :caption: Experiment Author Documentation
```

Comment:
I think the distinction between `Core Contributor Documentation` and `Experiment Author Documentation` here isn't clearly delineated. Maybe we need some higher up `Developing with Dallinger` section that includes `developing_dallinger_setup_guide`, `the_experiment_class`, `classes`, and `web_api`. Followed by separate sections about developing experiments and contributing back to the project. Again, this is a nitpick and this new organization is a clear improvement.

Reply:
I'm happy to keep evolving the docs in the pursuit of clarity for the end user :)
Perhaps for the next release?